src/cookies.rs

```diff
@@ -0,0 +1,48 @@
+use cookie::Cookie;
```

Comment:
`use cookie::{Cookie, CookieJar, ParseError};` or am I missing something?

Reply:
> `use cookie::{Cookie, CookieJar, ParseError};` or am I missing something?

I was lazy and left it to rustfmt. Looks like neither rustfmt nor CLion is that smart yet. I will fix it.